google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -132,7 +132,9 @@ def __init__(
             kwargs["client_options"] = {
                 "api_endpoint": os.environ.get("PUBSUB_EMULATOR_HOST")
             }
-            kwargs["credentials"] = AnonymousCredentials()
+            # Configure credentials directly to transport, if provided.
```

Comment:
Add a todo for https://github.com/googleapis/python-pubsub/issues/1349 here as well